app/code/PalashAs/AskQuestion/view/frontend/web/js/ask-question.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'jquery',
+    'jquery/ui',
+    'Magento_Ui/js/modal/alert',
```

Comment:
You're adding `alert` here, but not accepting it in the function parameters like ` function ($, alert) {`
Also, it should go before `jquery/ui` in this case